APM-Demo5/angular.json

```diff
@@ -123,5 +123,8 @@
       }
     }
   },
-  "defaultProject": "APM"
+  "defaultProject": "APM",
+  "cli": {
```

Comment:
I've been trying not to check in these analytics flags.

Reply:
Agree peoples privacy is beyond this course :)